recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "python-lsp-black" %}
-{% set version = "1.2.1" %}
+{% set version = "2.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d7eaeab2a377e96a82cc26afe2f8f2e1cf7c6eaefdcdeab026343e2e559dcce9
+  sha256: 8286d2d310c566844b3c116b824ada6fccfa6ba228b1a09a0526b74c04e0805f
 
 build:
   number: 0
-  skip: True  # [py<37]
-  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<38 or py>311]
```

Comment:
```suggestion
  skip: True  # [py<38]
```

Reply:
https://github.com/python-lsp/python-lsp-black/blob/v2.0.0/pyproject.toml#L2
This is just the target of their linting tool, but doesn't mean that this package doesn't work with 3.12.

Reply:
And they do test with 3.12, see https://github.com/python-lsp/python-lsp-black/blob/v2.0.0/.github/workflows/python.yml#L14.

Reply:
Fair enough! Thanks